core/templates/dev/head/app.js

```diff
@@ -374,7 +374,7 @@ oppia.config(['toastrConfig', function(toastrConfig) {
     messageClass: 'toast-message',
     progressBar: false,
     tapToDismiss: true,
-    timeOut: 1500,
+    timeOut: 4000,
```

Comment:
This affects **all** bottom-right toastr alerts, not just the one for solutions. Is this the intent? @ishucr7 @DubeySandeep

Reply:
In other places, this toastr cards timespans looks perfect to me (especially for those which were used in Learner's playlist service [here](https://github.com/oppia/oppia/blob/c820e0aeb976e1fbc09d966d51f555ce0587f2d8/core/templates/dev/head/domain/learner_dashboard/LearnerPlaylistService.js#L51)) I'm not sure why we need a 4 secs of timeout

Reply:
@DubeySandeep It is mentioned in the issue that's why i have changed it, and also the toast disappears very quickly. It might be of advantages to increase the timeout.

Reply:
If everything other than this seems fine as @DubeySandeep  said, then we should use some other implementation so that only this particular toast message gets affected.

Reply:
Can you guide me on this. @DubeySandeep

Reply:
@yashLadha In general, we discourage "guide me" questions without context, since they're hard to answer and generally result in someone else doing the actual work and investigation. If someone else is doing most of the investigation, then they should probably be the person assigned to the issue instead. Note that a large part of solving this issue consists of investigation; the actual coding is likely to be fairly minimal. 

In the future, please instead explain in detail what you have tried, and then ask a more specific question. (See [this comment](https://github.com/oppia/oppia/pull/4650#issuecomment-364776490) by Tony for an example.)

As a first step, you might want to take a look at the Toastr project on GitHub, and see if they provide a way to do what you want. It's also fine to pick up a different issue if this one is too difficult.

Reply:
Yes @seanlip, I will make sure of that.